docs/src/prototypes/chatPane/services/messageFactoryMock.tsx

```diff
@@ -34,20 +37,41 @@ const statusMap: Map<UserStatus, StatusPropsExtendable> = new Map([
 function generateChatMsgProps(msg: MessageData, fromUser: UserData): ChatMessage {
   const { content, mine } = msg
   const msgProps: ChatMessage = {
-    content: msg.withAttachment
-      ? { content: createMessageContentWithAttachments(content) }
-      : content,
+    // aria-labelledby will need to by generated based on the needs. Currently just hardcoded.
+    role: undefined,
```

Comment:
where and how this role is used?

Reply:
but I guess it's because by default we have set role="presentation" in behavior. Then probably it's needed to create an additional PR for improving ```messageBehavior``` to set role="undefined".
@jurokapsiar @kolaps33 need your input here

Reply:
discussed with @kolaps33, will update behavior in this PR (just remove "role" attribute)